hat/assets/js/apps/Iaso/domains/app/translations/fr.json

```
@@ -409,6 +412,7 @@
     "iaso.instance.titleSingle": "Soumission",
     "iaso.instance.undelete": "Marquer une(des) soumission(s) comme non-effacée(s)",
     "iaso.instance.unDeleteCount": "Marquer {count} soumission(s) comme non-effacée(s)",
+    "iaso.instance.unLinkReferenceSubmissionsConfirmMessage": "Il existe {linkedCount} soumissions de référence liées. Pour les dissocier, cochez la case correspondante.",
```

So that it matches the English text:
```suggestion
    "iaso.instance.unLinkReferenceSubmissionsConfirmMessage": "Vous avez sélectionné {linkedCount} soumissions de référence. Cochez cette case pour les dissocier de leur unité d'org. respective et les rétrograder en soumissions standard."
```